### PR TITLE
Solar Bankers is One Example of a Company Adopting the Revolutionary Skycoin Blockchain.de.md

### PR DESCRIPTION
**Pull request by Asgaror**

### CODE REVIEW

**Comment by Asgaror** [comment]
2cWSaq78ViRQBBeMA4RsLFPtw8QstbVCNiy

File: content/statement/Solar Bankers is One Example of a Company Adopting the Revolutionary Skycoin Blockchain.de.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+Diese Funktionalitäten machen Skycoin zur besten Wahl für die Entwicklung von Cryptowährungen. Da mehr und mehr Unternehmen auf Skycoins open-source Plattform setzen, verbreitet sich die Technologie rapide. Sehr wahrscheinlich wird Skycoins Technologie zukünftig Grundlage für verschiedenste Typen von Währungen sein.
+
+Obwohl die Verbreitung der Technologie von Skycoin gute Neuigkeiten für die Nutzer ist, ist es wichtig zu verstehen, dass auf skycoin-basierende Cryprowährungen unabhängig von dem Skycoin Projekt an sich sind. Da Skycoin open-source ist, kann die Technologie jeder verwenden. Viele Unternehmen nehmen sie und modifizieren sie für ihre eigenen Zwecke. Skycoin ist nicht mit diesen Projekten affiliiert. Stattdessen ist Skycoin das Fundament, für das sich diese Unternehmen entschieden haben.
```
Comment (Doigmann) [inline content/statement/Solar Bankers is One Example of a Company Adopting the Revolutionary Skycoin Blockchain.de.md]: ...für die Nutzer sind, ...

File: content/statement/Solar Bankers is One Example of a Company Adopting the Revolutionary Skycoin Blockchain.de.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+Als wir Skycoin entwicklet haben, hatten wir uns als Ziel gesetzt die beste Plattform seiner Art zu erschaffen. Die weitverbreitete Aneignung der Skycoin-Technologie spricht für sich selbst. Unternehmen wie Solar Bankers sind nur der Anfang. Es ist unsere Hoffnung, dass die Skycoin-Plattform Unternehmen die benötigten Tools bereitstellt, die sie für Innovation und Entwicklung benötigen - das Kreieren von neuen Technologien, welche die Nutzer ermächtigen und die dezentrale peer-to-peer Internetökonomie unterstützen.
```
Comment (Doigmann) [inline content/statement/Solar Bankers is One Example of a Company Adopting the Revolutionary Skycoin Blockchain.de.md]: ....weitverbreitete Annahme oder der weitverbreitete Einsatz...

**Comment by kietob** [comment]
Is there still worked on this document? Does anyone need some support to finish it?

**Pull request merged**